File: codeup/100000576/A.cpp
#include <iostream>
using namespace std;

int main() {
  int N, M, score, score_list[101];
  while (scanf("%d", &N) != EOF && N != 0) {
    fill(score_list, score_list + 101, 0);
    while (N--) {
      scanf("%d", &score);
      score_list[score]++;
    }
    scanf("%d", &M);
    printf("%d\n", score_list[M]);
  }

  return 0;
}

File: codeup/100000576/A.py
#!/usr/bin/python3
import sys

while True:
    line = sys.stdin.readline()
    if not line:
        break
    N = int(line)
    if N == 0:
        break
    score_list = [0 for x in range(101)]
    for score in input().split():
        score_list[int(score)] += 1
    M = int(input())
    print(score_list[M])
